apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -12,32 +23,49 @@ var connect = require('react-redux').connect;
 var ApplabVisualizationColumn = React.createClass({
   propTypes: {
     isReadOnlyWorkspace: React.PropTypes.bool.isRequired,
+    hideSource: React.PropTypes.bool.isRequired,
+    isEmbedView: React.PropTypes.bool.isRequired,
+
     isEditingProject: React.PropTypes.bool.isRequired,
     screenIds: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
     onScreenCreate: React.PropTypes.func.isRequired,
   },
 
   render: function () {
+    var showFrame = !this.props.isShareView;
+
     return (
-      <div id="visualizationColumn">
+      <div
+          id="visualizationColumn"
+          className={classNames({with_padding: this.props.visualizationHasPadding})}
+          style={[
+            (this.props.isEmbedView || this.props.hideSource) && styles.nonResponsive
+          ]}
```

Comment:
Meh.  I'm ambivalent.  This is a particularly noisy example (with both the expression and the style array inlined) but it doesn't seem too bad.

By the way, it's the array extraneous here?  Could read:
```javascript
style={(this.props.isEmbedView || this.props.hideSource) && styles.nonResponsive}
```